### PR TITLE
Changing CoNLL-X data format reference change as the previous URL is …

### PR DESCRIPTION
**Pull request by kaushikacharya**
…no more available.

Resolving issue https://github.com/dmcc/PyStanfordDependencies/issues/26

### CODE REVIEW

File: StanfordDependencies/CoNLL.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ class Token(namedtuple('Token', FIELD_NAMES_PLUS)):
     There are other fields but they typically won't be populated by
     StanfordDependencies. Fields are immutable.
 
-    See http://ilk.uvt.nl/conll/#dataformat for a complete description."""
+    See CoNLL-X shared task on Multilingual Dependency Parsing by Buchholz and Marsi(2006)
```
Comment (dmcc) [inline StanfordDependencies/CoNLL.py]: Micro-nit: can you wrap this text so it passes flake8? (pretty sure the failing nosetests are unrelated)

https://travis-ci.org/dmcc/PyStanfordDependencies/jobs/506774535#L692

**Comment by dmcc** [comment]
LGTM modulo minor formatting stuff!

File: StanfordDependencies/CoNLL.py (hunk 1 above)
Comment (kaushikacharya) [reply StanfordDependencies/CoNLL.py]: Committed the changes.

**Comment by dmcc** [comment]
Awesome, thank you!

**Pull request merged**